hera_sim/visibilities/simulators.py

```diff
@@ -220,6 +220,11 @@ def lsts(self):
         _, unique_inds = np.unique(self.uvdata.time_array, return_index=True)
         return self.uvdata.lst_array[unique_inds]
 
+    @cached_property
+    def times(self):
+        """Times at which data is defined."""
+        return np.unique(self.uvdata.time_array)
```

Comment:
Is `unique` still the best way to get unique times from pyuvdata? If we know the ordering and number of times and baselines, we can probably just slices the time_array. But this might be best implemented on pyuvdata.